metrics/prometheus.go

```diff
@@ -529,7 +529,7 @@ func (c *PrometheusCollector) collectContainersInfo(ch chan<- prometheus.Metric)
 		}
 
 		for k, v := range container.Spec.Labels {
-			baseLabels = append(baseLabels, sanitizeLabelName(k))
+			baseLabels = append(baseLabels, "label_"+sanitizeLabelName(k))
```

Comment:
Probably need to check for existing matching labels so we don't get any collisions?